home/constants/DateFormats.ts

```diff
@@ -0,0 +1,9 @@
+// https://date-fns.org/v2.11.0/docs/format
```

Comment:
love this idea - I will have to add a formats obj in all my projects in the future